### DISCUSSION TITLE
[Bug / Public Registry API] Download count of a package doesn't match sum of download count for all versions of that package

### DISCUSSION THREAD

**Discussion by zakyum** [community/community#134937] [npm]
Hello!

While working with the Public Registry API, I noticed that the sum of the ["Per version download counts" endpoint](https://github.com/npm/registry/blob/main/docs/download-counts.md#per-version-download-counts) doesn't match the download count of the ["Point values" endpoint](https://github.com/npm/registry/blob/main/docs/download-counts.md#point-values) for the same period.

It shouldn't matter if I get the total download count of a package via the point API or if I get the download count of all the versions and then manually sum them up because it's the same underlying data. So, at the end, I should get the same result for both, right? 

Well, to my surprise this is not the case, the sum of the download count per version is significantly higher, from 27% to 13%, for all packages which I tested. 

As I couldn't find anything regarding this in any of the docs pages or online, I consider this a bug.

Minimal reproducible ([also on JSFiddle](https://jsfiddle.net/wzb5u1mk/19/)):
```js
async function main() {
  const packages = ["express", "react", "yjs", "jquery", "lodash", "encodeurl"];
  for (const p of packages) {
    await compareDownloadsPerApiEndpoint(p);
    console.log();
  }
}

async function compareDownloadsPerApiEndpoint(packageName) {
  console.log(`Data for '${packageName}'`);

  const nDownloads = await fetch(`https://api.npmjs.org/downloads/point/last-week/${packageName}`)
    .then(response => response.json())
    .then(jsonBody => jsonBody.downloads);

  const sumOfAllVersions = await fetch(`https://api.npmjs.org/versions/${packageName}/last-week`)
    .then(response => response.json())
    .then(jsonBody => jsonBody.downloads)
    // get only the downloads as an array, without the versions
    .then(downloadsPerVersion => Object.values(downloadsPerVersion))
    // sum up the array of downloads
    .then(downloads => downloads.reduce((accumulator, currVal) => accumulator + currVal))

  console.log("Downloads last week, point API: ", nDownloads);
  console.log("Downloads last week, sum of versions: ", sumOfAllVersions);
  const diffInPercent = ((sumOfAllVersions / nDownloads) - 1) * 100;
  console.log(`Diff in percent: ${diffInPercent.toFixed(0)}%`);
}

main();
```

Hope you can help :)

UPDATE:
Seems like the issue has been noticed and fixed (works now). Although, an short answer/ack would have been nice...

**Comment by LiteBrite82** [comment]
@zakyum 
Sorry we missed that; I'm glad its resolved for you now.
